documentation/concepts/03A_transaction_fees.md

```diff
@@ -0,0 +1,80 @@
+---
+id: transactions
+title: Transactions Fees
+sidebar_label: Transaction Fees
+---
+
+# Transactions Fees
+
+A **transaction fee** is a fee that is required in order to process a transaction on the Aleo network. Currently, there are two types of transaction fees that correspond to each transaction type: **deploy** and **execute**.
+
+## Aleo Credits Demonination Table
+
+|Denomination|Size of Transaction|Value|Example|
+|:-:|:-:|:-:|:-:|
+|`microcredit`|Byte|1|Smallest denomination|
+|`millicredit`|Kilobyte (KB)|1000 microcredits|Most transactions average between 3 and 10 millicredits|
+|`credit`|Megabyte (MB)|1000 millicredits|Largest denomination|
+
+Note that transaction sizes are averages using [Big _O_ notation (on the order of)](https://en.wikipedia.org/wiki/Big_O_notation).
+
+# Types of Transaction Fees
+
+## Execute Transaction Fee
+A fee incurred when executing an application on the Aleo blockchain
+
+### Structure (optimistic)
+
+|Part of Execute Transaction|Size| Multiplier|Cost|
+|:-:|:-:|:-:|:-:|
+|`Transaction wrapper`|0 KB|1|0 millicredits|
+|`Transition wrapper`|0 KB|1|0 millicredits|
+|`Input`|1 KB|2|2 millicredits|
+|`Output`|1 KB|2|2 millicredits|
+|`Proof`|1 KB|1|1 millicredit|
+
+|Total|Size|Multiplier|Total cost|
+|:-:|:-:|:-:|:-:|
+|`Transaction -> Transition`|5 KB|1|5 millicredits|
+
+### Structure (pessimistic)
```

Comment:
"pessimistic" and "optimistic" might not be the right word. 

What about something like - `minimum and maximum` or `basic and packed` or some combination.

Reply:
Agreed, I struggled with this myself. Let's go with min and max.

Reply:
Changed to "minimum cost example" and "maximum cost example."